integrationTests/multiShard/endOfEpoch/startInEpoch/startInEpoch_test.go

```diff
@@ -213,6 +214,7 @@ func testNodeStartsInEpoch(t *testing.T, shardID uint32, expectedHighestRound ui
 	epochStartBootstrap, err := bootstrap.NewEpochStartBootstrap(argsBootstrapHandler)
 	assert.Nil(t, err)
 
+	_ = logger.SetLogLevel("*:DEBUG")
```

Comment:
Do you need this log to remain here?

Reply:
no but it was a good log to find the problem in the test.